### ISSUE TITLE
make:entity Repository already exists

### ISSUE THREAD

**Issue by devingray** [andersao/l5-repository#461]
When running php artisan make:entity SomeEntity 

There is an error that states repository already exists even though it doesn't

**Comment by eullercdr** [comment]
@devingray which version you are using? How are your version of Laravel?

**Comment by devingray** [comment]
Laravel 5.4, l5 version 2.6.24

**Comment by eullercdr** [comment]
@devingray Try again please

**Comment by devingray** [comment]
@eullercdr Still happening, upgraded to latest version. Perhaps at 5.4 we should go a few versions behind?

**Comment by eullercdr** [comment]
@devingray  I tested now version 2.6.25 error dont occurred. Verify again please. Wich command you are executing? Paste please.

**Comment by eullercdr** [comment]
@devingray Really the erro occurred .  Its very strange, because i can to create repositorys, but a time after, i cant, i am check that. You remember in which tag make:entity was working fine?

**Comment by Maras0830** [comment]
because this commit.


https://github.com/andersao/l5-repository/commit/b5ec10d5e762c05d57e6bcf5216fe95a7e747f26#diff-7f408a7932aa030a9844fa1817ebaf98

use 
```
php artisan make:entity SomeEntity
```

will be return 
```
Would you like to create a Validator? [y|N] (yes/no) [no]:
```

if answer yes, that will create Validator, but last code in EntityCommand.php

```
$this->call('make:repository', [
            'name'        => $this->argument('name'),
            '--fillable'  => $this->option('fillable'),
            '--rules'     => $this->option('rules'),
            '--validator' => $validator,
            '--force'     => $this->option('force')
        ]);
```
also created ... validator and throw FileAlreadyExistsException in RepositoryCommand.php, so if want to solve, suggest remove create validator when execute repository command.

**Comment by eullercdr** [comment]
Fixed in 2.6.25 thanks @Maras0830

**Comment by devingray** [comment]
Sorry didn't see these comments was away. Working perfectly! Legends
